src/index.js

```diff
@@ -301,6 +291,18 @@ export default class SauceLabs {
       args.push(`--proxy-sauce=${this.proxy}`);
     }
 
+    // Provide a default runner name. It's used for identifying the tunnel's initiation method.
+    let metadata = argv.metadata || '';
+    if (!metadata.includes('runner=')) {
+      metadata = [metadata, `runner=${DEFAULT_RUNNER_NAME}`]
+        .filter(Boolean)
+        .join(',');
+    }
+    args.push(`--metadata=${metadata}`);
+
+    const apiAddress = argv.apiAddress || ':8032';
```

Comment:
This one is problematic in the sense that it would prevent running two SC instances on the same host without specifying different `apiAddress` values. It's a normal constraint, just need to make sure it's gracefully handled with a clear error message. I remember it wasn't obvious what's wrong but maybe it's already fixed

Reply:
I will check that